src/gallium/drivers/lima/lima_draw.c

```diff
@@ -353,6 +353,12 @@ lima_pack_plbu_cmd(struct lima_context *ctx, const struct pipe_draw_info *info)
       ctx->buffer_state[lima_ctx_buff_pp_plb_rsw].size;
    plbu_cmd[i++] = 0x80000000 | (gl_position_va >> 4); /* RSW_VERTEX_ARRAY */
 
+   if (ctx->dirty & LIMA_CONTEXT_DIRTY_SCISSOR) {
```

Comment:
What if in the second frame, the scissor is enabled but not changed, so this won't be dirty but still need be set?

Reply:
Fixed, i found that we should use scissor field from rasterizer.
According to docs, it contains "Whether the scissor test is enabled."
and in case of multiple frames it's staying enabled till we disable it by glDisable.